components/sup/src/error.rs

```diff
@@ -161,7 +171,9 @@ impl fmt::Display for SupError {
             Error::EnvJoinPathsError(ref err) => format!("{}", err),
             Error::FileNotFound(ref e) => format!("File not found at: {}", e),
             Error::InvalidBinding(ref binding) => {
-                format!("Invalid binding - must be ':' delimited: {}", binding)
+                format!("Invalid binding \"{}\", must be of the form <NAME>:<SERVICE_GROUP> where \
```

Comment:
It's been helpful writing these error messages as their contents will be written out for humans to read and action against--I was one of those early humans in development :smile: